app/controllers/users_controller.rb

```diff
@@ -1,17 +1,8 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  before_action :find_user, except: :index
-
-  def index
-    @users = User.page(params[:page])
-  end
-
-  def destroy
-    @user.destroy
-    flash[:danger] = 'User profile has been deleted'
-    redirect_to users_path
-  end
+  include Pundit
```

Comment:
is already included in `ApplicationController`

Reply:
done